packages/styles/scss/components/lightbox-media-viewer/_lightbox-media-viewer.scss

```diff
@@ -38,6 +39,7 @@
         @include carbon--type-style('body-long-02');
 
         color: $text-02;
+        text-align: justify;
```

Comment:
Why did we add this? I don't think that's part of the design.

![image](https://user-images.githubusercontent.com/3793636/72557007-7c827c00-3865-11ea-8f66-5d31dfa63a6c.png)

Reply:
I have the text-align property to remove the end of the spaces of each line

Reply:
What space at the end of the line? Can you give an example so I'm on the same page? If it's the space I think you're talking about. That's supposed to be there. In general, we don't use justify here because it causes readability problems for people with dyslexia because of the extra space or rivers it creates within the text. The design is just aligned left and is fine with the rags I believe.